src/Login/Login.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+
+class Login extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			email: '',
+			password: '',
+			err: ''
+		};
+
+		this.update = this.update.bind(this);
+
+		this.displayLogin = this.displayLogin.bind(this);
+	}
+
+	update(e) {
+		let name = e.target.name;
+		let value = e.target.value;
+		this.setState({
+			[name]: value
+		});
+	}
+
+	displayLogin(e) {
```

Comment:
function names should be intuitive. It is hard to predict what this function does by looking at the name displayLogin.